runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -86,8 +81,8 @@
      */
     private final ConcurrentMap<String, FileDescriptor> fileDescriptorMap = new ConcurrentHashMap<>();
 
-    public DynamicProtobufSerializer(byte type, CorfuRuntime corfuRuntime) {
-        this.type = type;
+    public DynamicProtobufSerializer(CorfuRuntime corfuRuntime) {
+        this.type = ProtobufSerializer.PROTOBUF_SERIALIZER_CODE;
```

Comment:
is type needed in this class anymore?

Reply:
The getType is an interface method and needs to be implemented.
I'll leave this for uniformity.